test-utils/Cargo.toml

```diff
@@ -15,6 +15,6 @@ hyper = { version = "0.14.10", features = ["full"] }
 log = "0.4"
 serde = { version = "1", default-features = false, features = ["derive"] }
 serde_json = "1"
-soketto = "0.6"
+soketto = { version = "0.7", features = ["http"] }
```

Comment:
Do you think we should upgrade `soketto` in other places as well?

Reply:
yeah, we probably should. It just needs some refactoring because the soketto API now requires `httparse::Header` instead of `&str`